scripts/pre_commit_linter.py

```diff
@@ -1806,23 +2247,152 @@ def _check_html_directive_name(self):
-            else:
-                summary_message = '%s   HTML directive name check passed' % (
-                    _MESSAGE_TYPE_SUCCESS)
-                summary_messages.append(summary_message)
+            if failed:
+                summary_message = '%s   HTML directive name check failed' % (
+                    _MESSAGE_TYPE_FAILED)
+                summary_messages.append(summary_message)
+            else:
+                summary_message = '%s   HTML directive name check passed' % (
+                    _MESSAGE_TYPE_SUCCESS)
+                summary_messages.append(summary_message)
+
+            print ''
+            if total_files_checked == 0:
+                if self.verbose_mode_enabled:
+                    print 'There are no files to be checked.'
+            else:
+                print '(%s files checked, %s errors found)' % (
+                    total_files_checked, total_error_count)
+                print summary_message
+
+        return summary_messages
+
+
+class OtherLintChecksManager(LintChecksManager):
+    """Manages all the linting functions except the ones against Js and Ts. It
+    checks Python, CSS, and HTML files.
+
+    Attributes:
+        all_filepaths: list(str). The list of filepaths to be linted.
+        py_filepaths: list(str). The list of python filepaths to be linted.
+        html_filepaths: list(str). The list of html filepaths to be linted.
+        css_filepaths: list(str). The list of css filepaths to be linted.
+        other_filepaths: list(str). The list of other filepaths to be linted.
+        verbose_mode_enabled: bool. True if verbose mode is enabled.
+    """
+    def __init__(
+            self, py_filepaths, html_filepaths, css_filepaths, other_filepaths,
+            verbose_mode_enabled=False):
+        """Constructs a OtherLintChecksManager object.
+
+        Args:
+            py_filepaths: list(str). The list of python filepaths to be linted.
+            html_filepaths: list(str). The list of html filepaths to be linted.
+            css_filepaths: list(str). The list of css filepaths to be linted.
+            other_filepaths: list(str). The list of other filepaths to be
+                linted.
+            verbose_mode_enabled: bool. True if verbose mode is enabled.
+        """
+        self.py_filepaths = py_filepaths
+        self.html_filepaths = html_filepaths
+        self.other_filepaths = other_filepaths
+        self.css_filepaths = css_filepaths
+        self.all_filepaths = (
+            py_filepaths + html_filepaths + other_filepaths + css_filepaths)
+        LintChecksManager.__init__(
+            self, self.all_filepaths, verbose_mode_enabled)
+
+    def _lint_all_files(self):
+        """This function is used to check if node-eslint dependencies are
+        installed and pass ESLint binary path and lint all the files(JS, Python,
+        HTML, CSS) with their respective third party linters.
+        """
+
+        print 'Starting linter...'
```

Comment:
Note here that it is for CSS, HTML, and Python files.

Reply:
Done